examples/configs/homography_estimation.yaml

```diff
@@ -0,0 +1,14 @@
+profiling: false
```

Comment:
Remember to update the examples/readme file with the details for profiling in addition to the homography details already there.